OpenRA.Game/GameInformation.cs

```diff
@@ -114,7 +114,7 @@ public void AddPlayer(OpenRA.Player runtimePlayer, Session lobbyInfo)
 				Color = runtimePlayer.Color,
 				Team = client.Team,
 				SpawnPoint = runtimePlayer.SpawnPoint,
-				IsRandomFaction = runtimePlayer.Country.Race != client.Country,
+				IsRandomFaction = runtimePlayer.Country.Race != client.CountryRace,
```

Comment:
Oh, I see.  This is a historical naming fail: `client.Country` actually specifies a race (and probably wants to be renamed to `Race`).